app/views/reports/regions/show.html.erb

```diff
@@ -116,9 +124,15 @@
           </p>
           <p class="m-0px <% if @data.last_value(:uncontrolled_patients_rate) %>c-black<% else %>c-grey-medium<% end %>">
             <% if @data.last_value(:uncontrolled_patients_rate) %>
-              <span class="fw-bold">
-                <%= number_with_delimiter(@data.last_value(:uncontrolled_patients)) %>
-                </span> of <%= number_with_delimiter(@data.last_value(:adjusted_registrations)) %> patients
+              <span class="fw-bold"><%= number_with_delimiter(@data.last_value(:uncontrolled_patients)) %></span> of <%= number_with_delimiter(@data.last_value(:adjusted_registrations)) %> patients
+              <span
+                data-toggle="tooltip"
+                data-placement="top"
+                data-trigger="hover"
+                title="Patients registered until the end of <%= @data[:adjusted_registrations].keys[-4] %>"
```

Comment:
Can you extract this to an instance variable in the controller, and then use that in the views?  Grabbing it with`-4` everytime is a little awkward.